management-api-common/src/main/java/com/datastax/mgmtapi/shims/CassandraAPI.java

```diff
@@ -82,4 +83,11 @@ default Object handleRpcResult(Callable<Object> rpcResult) throws Exception {
   default List<String> getKeyspaces() {
     return StorageService.instance.getKeyspaces();
   }
+
+  default void reloadInternodeEncryptionTruststore() throws Exception {
+    throw new Exception("Unimplemented for Cassandra, only available for DSE");
+  };
+  default X509Certificate[] getEncryptionTruststoreIssuers() throws Exception {
```

Comment:
suggestion:
Shoudl probably also change this to `UnsupportedOperationException` as well